ash/src/entry.rs

```diff
@@ -149,6 +149,23 @@ impl Entry {
         let entry_fn_1_0 = vk::EntryFnV1_0::load(load_fn);
         let entry_fn_1_1 = vk::EntryFnV1_1::load(load_fn);
 
+        Self::from_parts_1_3(
+            static_fn,
+            entry_fn_1_0,
+            entry_fn_1_1,
+            entry_fn_1_2,
+            entry_fn_1_3,
+        )
+    }
+
+    #[inline]
+    pub unsafe fn from_parts_1_3(
+        static_fn: vk::StaticFn,
+        entry_fn_1_0: vk::EntryFnV1_0,
+        entry_fn_1_1: vk::EntryFnV1_1,
+        entry_fn_1_2: vk::EntryFnV1_2,
+        entry_fn_1_3: vk::EntryFnV1_3,
```

Comment:
These are empty and unused arguments.

Reply:
Hmm, should we name it `_1_1`, or rename it in the future even when it doesn't change? I lean towards the former.

Reply:
remaed to `1_1`